src/components/WalletModal/index.tsx

```diff
@@ -129,16 +130,19 @@ const sendAnalyticsEventAndUserInfo = (
   account: string,
   walletType: string,
   chainId: number | undefined,
-  isReconnect: boolean
+  isReconnect: boolean,
+  peerWalletAgent: string | undefined
 ) => {
   sendAnalyticsEvent(InterfaceEventName.WALLET_CONNECT_TXN_COMPLETED, {
     result: WalletConnectionResult.SUCCEEDED,
     wallet_address: account,
     wallet_type: walletType,
     is_reconnect: isReconnect,
+    peer_wallet_name: peerWalletAgent,
```

Comment:
Should this now read `peer_wallet_agent`?
```suggestion
    peer_wallet_agent: peerWalletAgent,
```